locksmith/package.json

```diff
@@ -54,6 +54,7 @@
     "@types/supertest": "2.0.8",
     "@typescript-eslint/eslint-plugin": "2.3.0",
     "@unlock-protocol/unlock-js": "0.4.0",
+    "apollo-server": "^2.9.7",
```

Comment:
Probably remove the carets here, as our current process for deps makes them not working as expected :/